pkg/internal/controller/controller.go

```diff
@@ -307,7 +318,24 @@ func (c *Controller) reconcileHandler(ctx context.Context, obj interface{}) {
 		return
 	}
 
-	log := c.Log.WithValues("name", req.Name, "namespace", req.Namespace)
+	// Add object to the logger.
+	var objectLogKey = "obj"
+	if c.GroupVersionKind != nil {
+		objectLogKey = strings.ToLower(c.GroupVersionKind.Kind)
```

Comment:
Note to myself:
* We should follow the convention for keys too: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments (affects kind + at least reconciler group/kind)